staging_docs/user/guides/04-cache.md

```diff
@@ -0,0 +1,114 @@
+# Cache rubygems.org 
```

Comment:
Is that "build a caching proxy (of rubygems.org)"?

Reply:
Maybe?